profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -273,83 +265,62 @@ function _nexteuropa_varnish_purge_paths($paths) {
     $escaped_paths
   );
 
-  // Fill in placeholders in the request skeleton.
-  $values = array(
-    'host' => '',
-    'base_path' => '',
-    'clear_type' => 'regexp-multiple',
-    'path_regexp' => '^(' . implode('|', $escaped_paths) . ')$',
-    'tag' => $cache->vget('tag', ''),
-  );
+  $request_method = variable_get('nexteuropa_varnish_request_method', 'PURGE');
+  $request_path = '/invalidate';
 
-  $request = $cache->defaultRequestSkeleton();
-  array_walk_recursive(
-    $request,
-    function (&$item, $key, $values) {
-      foreach ($values as $token => $value) {
-        $item = str_replace('@{' . $token . '}', $value, $item);
-      }
-    },
-    $values
+  $base_request_headers = _nexteuropa_varnish_base_request_headers();
+  $request_headers = $base_request_headers + array(
+    'X-Invalidate-Tag' => variable_get('nexteuropa_varnish_tag'),
+    'X-Invalidate-Type' => 'regexp-multiple',
+    'X-Invalidate-Regexp' => '^(' . implode('|', $escaped_paths) . ')$',
   );
 
-  $http_targets = $cache->vget('http_targets', array());
-  $method = $request['method'];
-
-  // Remove empty headers.
-  $request['headers'] = array_filter(
-    $request['headers'],
-    function ($value) {
-      return drupal_strlen(trim($value)) > 0;
-    }
-  );
+  $http_targets = variable_get('nexteuropa_varnish_http_targets', array());
 
   foreach ($http_targets as $target) {
-    // First chance to customize the request.
-    if (function_exists('fp_alter_request_for_cache_page')) {
-      $request = fp_alter_request_for_cache_page(NULL, NULL, $target, $request);
-    }
-
-    // Format headers.
-    $formatted_headers = array_map(
-      function ($name, $value) {
-        return sprintf('%s: %s', $name, trim($value));
-      },
-      array_keys($request['headers']),
-      $request['headers']
+    $url = sprintf('http://%s%s', $target, $request_path);
+
+    $response = drupal_http_request(
+      $url,
+      array(
+        'headers' => $request_headers,
+        'method' => $request_method,
+        'timeout' => variable_get('nexteuropa_varnish_http_timeout', 2.0),
```

Comment:
the variable "nexteuropa_varnish_http_timeout" is not set before this call or is not documented for the additional settings.